_rules/__tests__/headings.js

```diff
@@ -22,14 +22,36 @@ const getAllHeadingsFromMarkdownBody = body => {
 	}, [])
 }
 
+/**
+ * Extract a number from a given string
+ * @param {String} str given string
+ * @returns {Number}
+ */
 const extractNumberFromGivenString = (str) => {
-	return str.match(/\d+/)[0]
+	const match = str.match(/\d+/)[0]
+	return parseInt(match)
 }
 
+/**
+ * Check if a given array has duplicates
+ * @param {Array<Object>} array arr
+ * @returns {Boolean}
+ */
 const arrayHasDuplicates = (array) => {
```

Comment:
This is not needed anymore since strict increase by one will ensure no duplicates.